src/lint/rules/tests/indentation/in9.rs

```diff
@@ -4,9 +4,9 @@ pub static IN9_CORRECT_CASE_INDENT: &str = "
 method some_switch(int arg) {
     switch(arg) {
     case ZERO:
-#if (asdd == 0) {
+    #if (asdd == 0) {
         some_call();
-}
+    }
         if (a) {
```

Comment:
Leave the #if statement and its closing brace aligned to the first column.
Lets consider them as preprocessor commands like C macros.

Reply:
Can be done later